README.md

```diff
@@ -6,10 +6,17 @@
 
 ```groovy
    repositories {
-       maven()
+      maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+        maven { url "https://dl.bintray.com/ethereum/maven" }
```

Comment:
don think we need the second repo

Reply:
it will throw an error otherwise something like this 
```
Required by:
    project : > org.web3j:web3j-unit:4.6.0-SNAPSHOT:20191009.065428-2
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':test'.
> Could not resolve all files for configuration ':testRuntimeClasspath'.
   > Could not find org.ethereum:solcJ-all:0.4.25.
```

Reply:
ok